doc/user/events/widget.rst

```diff
@@ -304,8 +304,51 @@ Hosted or pretix Enterprise are active, you can pass the following fields:
 * If you use the campaigns plugin, you can pass a campaign ID as a value to ``data-campaign``. This way, all orders
   made through this widget will be counted towards this campaign.
 
-* If you use the tracking plugin, you can pass a Google Analytics User ID to enable cross-domain tracking. This will
-  require you to dynamically load the widget, like this::
+* If you use the tracking plugin, you can enable cross-domain tracking. To do so, you need to dynamically build 
```

Comment:
"dynamically build" sounds like you need to compile something, maybe stick with "load" or use "initialize"?

Reply:
I guess „initialize“ is the word. „load“ for me somehow made it unclear, whether I still need the html-code from the top of the page. What about „you need to initialize the pretix-widget manually“?

Reply:
:+1: